students/minki-choi/models/DrinksDao.js

```diff
@@ -0,0 +1,38 @@
+import prisma from '../../../prisma'
+
+const findDrinks = async () => {
+  const drinks = await prisma.$queryRaw(`
+    SELECT 
+      drinks.id, drinks.korean_name, drinks.english_name, categories.name, nutritions.kcal, nutritions.sodium, nutritions.fat, nutritions.protein, nutritions.sugars, nutritions.caffeine, allergies.name
+    FROM categories
+    JOIN drinks
+      ON categories.id = drinks.category_id
+    JOIN nutritions
+      ON drinks.id = nutritions.drink_id
+    LEFT JOIN drink_allergies
+      ON drinks.id = drink_allergies.drink_id
+    LEFT JOIN allergies
+      ON allergies.id = drink_allergies.allergy_id
+  `)
```

Comment:
```suggestion
  const drinks = await prisma.$queryRaw(`
    SELECT 
      drinks.id,
      drinks.korean_name, 
      drinks.english_name, 
      categories.name, 
      nutritions.kcal, 
      nutritions.sodium, 
      nutritions.fat, 
      nutritions.protein, 
      nutritions.sugars, 
      nutritions.caffeine, 
      allergies.name
    FROM categories
    JOIN drinks
      ON categories.id = drinks.category_id
    JOIN nutritions
      ON drinks.id = nutritions.drink_id
    LEFT JOIN drink_allergies
      ON drinks.id = drink_allergies.drink_id
    LEFT JOIN allergies
      ON allergies.id = drink_allergies.allergy_id
  `)
```
이렇게 하는 것이 읽기 더 편하겠네요 :)

Reply:
수정되었습니다. 감사합니다.